bentoml/_internal/io_descriptors/json.py

```diff
@@ -241,14 +241,16 @@ async def from_http_request(self, request: Request) -> JSONType:
                 pydantic_model = self._pydantic_model.parse_obj(json_obj)
                 return pydantic_model
             except pydantic.ValidationError as e:
-                raise BadInput(f"Invalid JSON input received: {e}") from None
+                raise BadInput(f"Invalid JSON input received: {e}") from e
```

Comment:
Hm, do you know why this was `None` before?

Reply:
no idea.

Reply:
@sauyon @aarnphm https://peps.python.org/pep-0415/ that was for better exception context